src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -117,13 +118,13 @@
     /**
      * The thread that expires {@link JitsiMeetConference}s.
      */
-    private FocusExpireThread expireThread = new FocusExpireThread();
+    private final FocusExpireThread expireThread = new FocusExpireThread();
 
     /**
      * Jitsi Meet conferences mapped by MUC room names.
      */
-    private Map<String, JitsiMeetConference> conferences
-        = new HashMap<String, JitsiMeetConference>();
+    private final Map<String, JitsiMeetConference> conferences
+        = new ConcurrentHashMap<>();
```

Comment:
Yes, it is because of getConferenceCount(). I was reluctant to synchronize on the FocusManager object. If all synchronization on the FocusManager object is solely for the purpose of protecting #conferences, then I would rather synchronize on #conferences instead everywhere. But I wasn't able to convince myself that this is the case (e.g. enableGracefulShutdownMode() seems to need it because of #shutdownInProgress).